guides/keras_nlp/getting_started.py

```diff
@@ -486,8 +500,8 @@ def preprocess(inputs, label):
 pretraining_model.summary()
 pretraining_model.compile(
     loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
-    optimizer=keras.optimizers.experimental.AdamW(learning_rate=5e-4),
-    weighted_metrics=keras.metrics.SparseCategoricalAccuracy(),
+    optimizer=keras.optimizers.AdamW(learning_rate=5e-4),
```

Comment:
Isn't this just `Adam` now with the `weight_decay` parameter set?

Reply:
Interesting! I didn't know this. But it is still exported as a separate name.

Since `AdamW` will work both in `tf.keras` and `keras-core`, that's probably the safe way to leave this for now.

Reply:
IIUC AdamW is *decoupled* weight decay. There's a bunch of sthick in the [paper](https://arxiv.org/pdf/1711.05101.pdf) about how you have to be very careful adding weight decay in Adam.

Reply:
I think what @ianstenbit is pointing out is that https://github.com/keras-team/keras-core/blob/main/keras_core/optimizers/adamw.py is literally just an empty subclass. Not sure if that reflects a bug in `AdamW` in keras-core, or if `keras_core.optimizers.Adam` is correctly handling the decoupled weight decay.